thirdeye-ui/src/app/components/rca/top-contributors-section/top-contributors-section.component.tsx

```diff
@@ -0,0 +1,242 @@
+/*
+ * Copyright 2023 StarTree Inc
+ *
+ * Licensed under the StarTree Community License (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at http://www.startree.ai/legal/startree-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT * WARRANTIES OF ANY KIND,
+ * either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+import Box from "@material-ui/core/Box";
+import Button from "@material-ui/core/Button";
+import Grid from "@material-ui/core/Grid";
+import Typography from "@material-ui/core/Typography";
+import every from "lodash/every";
+import isEmpty from "lodash/isEmpty";
+import React, { FunctionComponent, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useOutletContext, useParams } from "react-router-dom";
+import { NoDataIndicator } from "../../no-data-indicator/no-data-indicator.component";
+import { EmptyStateSwitch } from "../../page-states/empty-state-switch/empty-state-switch.component";
+import { LoadingErrorStateSwitch } from "../../page-states/loading-error-state-switch/loading-error-state-switch.component";
+import { AnomalyFilterOption } from "../anomaly-dimension-analysis/anomaly-dimension-analysis.interfaces";
+import { PreviewChart } from "../top-contributors-table/preview-chart/preview-chart.component";
+import { TopContributorsTable } from "../top-contributors-table/top-contributors-table.component";
+import {
+    SkeletonV1,
+    useNotificationProviderV1,
+} from "../../../platform/components";
+import { ActionStatus } from "../../../rest/actions.interfaces";
+import {
+    AnomalyDimensionAnalysisData,
+    SavedStateKeys,
+} from "../../../rest/dto/rca.interfaces";
+import { areFiltersEqual } from "../../../utils/anomaly-dimension-analysis/anomaly-dimension-analysis";
+import { getFromSavedInvestigationOrDefault } from "../../../utils/investigation/investigation.util";
+import { notifyIfErrors } from "../../../utils/notifications/notifications.util";
+import { serializeKeyValuePair } from "../../../utils/params/params.util";
+import { RootCauseAnalysisForAnomalyPageParams } from "../../../pages/root-cause-analysis-for-anomaly-page/root-cause-analysis-for-anomaly-page.interfaces";
+import { InvestigationContext } from "../../../pages/rca/investigation-state-tracker-container-page/investigation-state-tracker.interfaces";
+import { TopContributorsSectionProps } from "./top-contributors-section.interfaces";
+import {
+    Accordion,
+    AccordionDetails,
+    AccordionSummary,
+} from "@material-ui/core";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+
+export const TopContributorsSection: FunctionComponent<TopContributorsSectionProps> =
```

Comment:
This is the renamed [#1360 (files)](https://github.com/startreedata/thirdeye/pull/1360/files#diff-cf1432596f3100381e2821f310da3e719c13951c0730f31044cb48ccd4b0513c) file that is shown as deleted below